src/content/docs/infrastructure/host-integrations/host-integrations-list/lightptd-integration.mdx

```diff
@@ -0,0 +1,176 @@
+
+```
+server.modules = (
+    "mod_1indexfile",
+    "mod_access”,
+    "mod_alias",
+    "mod_redirect”,
+    "mod_status"
+)
+
+server.document-root	= "/var/www/html/1lighttpd-webpage"
+server.upload-dirs	  = ( "/var/cache/lighttpd/uploads" )
+server.errorlog		    = "/var/log/lighttpd/error.log"
+server.pid-file		    = "/run/lighttpd.pid"
+server.username		    = "www-data"
+server.groupname	    = "www-data"
+server.port		        = 9880
+status.status-url	    = "/server-status”
+```
+</Step>
+
+
+<Step>
+
+## Nri-Flex configuration
+
+First, follow the [installation steps](https://github.com/newrelic/nri-flex/blob/master/docs/basic-tutorial.md) of nri-flex from [nri-flex repository](https://github.com/newrelic/nri-flex)
+
+Once you have installed the infrastructure agent on your host, nri-flex binary is also installed with it.
+
+To create a flex configuration file follow these steps:
+
+1. Path:
+```
+/etc/newrelic-infra/integrations.d
+```
+
+2. Create a file with the name `lighttpd-http-config.yml`
+
+Note: A generic file name could be &ltFILENAME&gt-config.yml
+
+3. Use add the following configuration to your `lighttpd-http-config.yml` file. 
+
+Sample configuration file: https://github.com/newrelic/nri-flex/blob/master/examples/lighttpd-http-example.yml
+
+It's important to not that you can replace `base_url` and `url` with your application url (Please refer to url documentation for more understanding) and also change url in the `commands` attribute. You can also convert the metrics format which you get through url into regex and add expressions in regex_matches. EVENT_TYPE is also adjustable. 
+
+Once your configuration file is updated, it will look like this:
+
+```
+---
+integrations:
+  - name: nri-flex
+    # interval: 30s
+    config:
+      name: lighttpdFlex
+      global:
+        base_url: http://lighttpd.net/
+      apis:
+        - event_type: status
+          url: status?format=plain
+        - event_type: &lTHE_PLACE_WHERE_METRICS_ARE_CAPTURED&gt
+          commands:
+            - cache: status?format=plain
+              split_by: ": "
+          snake_to_camel: true
+
+
+  - name: nri-flex
+    interval: 10s
+    # timeout: 5s
+    config:
+      name: test
+      apis:
+        - event_type: &ltTHE_PLACE_WHERE_METRICS_ARE_CAPTURED&gt
+          commands:
+            - run: curl 'http://lighttpd.net/status?auto'
+              split_by: ": "
+              regex_matches:
+                - expression: (\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)
+```
+
+</Step>
+
+<Step>
+
+### Restart the New Relic infra agent and lighttpd service.
+
+To restart the New Relic infra agent follow the instructions in [this doc](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) and run this command:
```

Comment:
I'd write this: 
To restart the New Relic infra agent follow [these instructions](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) and run this command:

Reply:
better! thanks!